repos/system_upgrade/el7toel8/actors/peseventsscanner/libraries/library.py

```diff
@@ -97,6 +112,11 @@ def get_transaction_configuration():
     return transaction_configuration
 
 
+def filter_releases_by_target(releases, target):
+    return [r for r in releases if version.matches_version(
+        ['<= {}.{}'.format(*target)], '{}.{}'.format(*r))]
```

Comment:
nit pick for future reference, this is through out all iterations the same so it could be made a variable before the loop. But this is not going to be too huge, therefore not needed to change that now. Just wanted to point that out.

Reply:
Good point, changed it now. ;)